sphinxcontrib/datatemplates/cli.py

```diff
@@ -36,24 +24,45 @@ def main():
         'template',
         help='the path to the template file',
     )
+    parser.add_argument(
+        '--option',
```

Comment:
```suggestion
        '--option', '-o',
```

I think handling the options this way is fine to start. We could do something smarter for the well-known options used by each loader later.